src/Whoops/Util/TemplateHelper.php

```diff
@@ -59,6 +63,42 @@ public function escapeButPreserveUris($raw)
     }
 
     /**
+     * Format the given value into a human readable string.
+     *
+     * @param  mixed $value
+     * @return string
+     */
+    public function dump($value)
+    {
+        if (class_exists('Symfony\Component\VarDumper\Cloner\VarCloner')) {
+            $cloner = new VarCloner();
+            if ('cli' === PHP_SAPI) {
```

Comment:
> The entire internet of PHP devs use this

Not true, really. I'd never use it in that context. I'd always have that part at a higher level of abstaction.